src/app/user/(logged-in)/my-account/edit/_components/account-edit-form.tsx

```diff
@@ -75,7 +75,7 @@ export default function AccountEditForm({
             { label: "No", value: USER_RESIDENCY.OTHER },
           ]}
         />
-        <div className="mt-6 flex gap-2">
+        <div className="mt-6 flex flex-col gap-2 sm:flex-row">
```

Comment:
Currently the breakpoint I have used to switch to "big screen" is `md` and I have used this everywhere. I don't have a reason for using `md`, other than it "felt right", which is a terrible reason given it is greatly biased towards my small laptop screen and poor eyesight.

I noticed in this PR you have elected to `sm` as the breakpoint, here an in the button variants. Is there a reason for this?

Reply:
![Screenshot 2024-05-04 at 11 40 56](https://github.com/barkbank/barkbank-pawtal/assets/48792128/a963f093-e5b7-4acd-a9d3-6329fa84fc6d)
![Screenshot 2024-05-04 at 11 41 16](https://github.com/barkbank/barkbank-pawtal/assets/48792128/2cc1f2bb-970b-4c29-bd2d-0fd0efae043f)

I think my decision is purely a "feels right" kind of decision 😅😂 without any specific UIUX considerations. I have however scanned through the tailwind docs to find that `sm` is the breakpoint for tablets which would "look better" without a full-widthed button.

Reply:
I will change it to `md` breakpoints for standardisation across the entire app!

Reply:
Done !
4c9d214560ba78e5d7688561c15af8e4d13edb8b
a65ad9d97242f7ba455195835f197e81f3257a50

Reply:
Nice. I added [a ticket](https://www.notion.so/MAYBE-Define-custom-breakpoint-fcf230747c0947d982cabf9a87a5ad0f?pvs=4) for considering defining a custom breakpoint. We can discuss further.